tests/test_common_gpu.py

```diff
@@ -1083,6 +1084,50 @@ def test_8bit_dora_merging(self):
         assert torch.allclose(out_dora, out_unmerged, atol=atol, rtol=rtol)
         assert torch.allclose(out_dora, out_unloaded, atol=atol, rtol=rtol)
 
+    @require_torch_gpu
+    @pytest.mark.single_gpu_tests
+    @require_bitsandbytes
+    def test_8bit_dora_cpu_offloading(self):
+        torch.manual_seed(0)
+        model = AutoModelForCausalLM.from_pretrained(
+            "facebook/opt-125m",
+            quantization_config=BitsAndBytesConfig(load_in_8bit=True),
```

Comment:
Could you explain why we need to use bnb here?

Reply:
We don't. I just used one of the existing tests as starting point. I can drop that part for sure.

Reply:
All right, please drop bnb, let's keep the test focused on what is needed.